README.md

```diff
@@ -4,7 +4,7 @@
 
 There is a single [step](https://jenkins.io/doc/pipeline/steps/ws-cleanup) to be used whenever workspace is allocated. 
 
-## Job DSL
+### Scripted pipeline (Job DSL)
```

Comment:
Scripted Pipeline and Job DSL are two different concepts. I suggest leaving this section as-is and adding a new section covering the use of `preBuildCleanup` with Scripted Pipeline.